theme/styles/templates/page.scss

```diff
@@ -0,0 +1,170 @@
+.Page {
+  h1, h2, h3, p, ol, ul {
+    font-family: $itc-garamond;
+  }
+
+  h1, h2, h3, h4, h5, h6 {
+    padding-bottom: 3.125rem;
+  }
+
+  h2 {
+    @extend %serif-sm;
+
+    a {
+      @extend %button--style-primary;
+      font-family: $gt-zirkon;
+      font-size: 2.5rem;
+      line-height: 2.625rem;
+      
+      @include media('md-up') {
+        font-size: 4.0625rem;
+        line-height: 3.65625rem;
+      }
+    }
+
+    @include media('md-up') {
+      font-size: 4.6875rem;
+      line-height: 4.6875rem;
+    }
+  }
+  
+  h3, p, ol, ul {
+    @extend %serif-md;
+  }
+
+  h4, h5 {
+    font-family: $gt-zirkon;
+  }
+
+  h4 {
+    @extend %sans-md;
+    text-transform: uppercase;
+  }
+
+  h5 {
+    @extend %sans-sm;
+  }
+
+  p, ol, ul {
+    margin-bottom: 2rem;
+
+    @include media('md-up') {
+      margin-bottom: 5.625rem;
+    }
+
+    a {
+      &:hover {
+        color: lighten(color('black'), 25%);
+      }
+    }
+  }
+
+  ol {
+    margin: 2rem;
+    margin-right: 0;
+  }
+
+  ul {
+    margin: 2rem;
+    margin-right: 0;
+  }
+
+  table {
+    display: block;
+    margin: 0 auto;
+    overflow-x: auto;
+    white-space: nowrap;
+    border-collapse: collapse;
+    border-top: 1px solid color('black');
+
+    tr {
+      &:not(last-of-type) {
+        border-bottom: 1px solid color('black');
+      }
+
+      td {
+        min-width: calc(50vw - 1.25rem);
+        vertical-align: middle;
+        padding: 1.25rem 0;
+
+        @include media('md-up') {
+          padding: 1.6875rem 0;
+          min-width: calc(50vw - 2.5rem);
```

Comment:
can we use the sizing variables you declared earlier?

Reply:
YEs! Forgot about this one!